scripts/deploy

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# This workflow builds and deploys the master-branch to the production
+# web server. Deployment is done using rsync over SSH, only uploading
+# files that have changed since the last deployment.
+#
+# `DEPLOY_USER`
+#    User account used to connect via SSH to the web server
+#
+# `DEPLOY_USER_PRIVATE_KEY`
+#    Base64 encoded private key of the user specified in `DEPLOY_USER`
+#
+# `DEPLOY_TARGET`
+#    Host of the production server
+#
+# `DEPLOY_TARGET_PUBLIC_KEY`
+#    Base64 encoded public key of the web server
+
+deploy_exclude="scripts/deploy-exclude"
+deploy_include="scripts/deploy-include"
+deploy_path="~/100eyes"
+
+# Write master.key
+echo $RAILS_MASTER_KEY > config/master.key
+
+# Add user's private key
+mkdir $HOME/.ssh
+echo $DEPLOY_USER_PRIVATE_KEY | base64 --decode > $HOME/.ssh/deploy_rsa
+chmod 400 $HOME/.ssh/deploy_rsa
+
+# Add target's key
+echo $DEPLOY_TARGET_PUBLIC_KEY | base64 --decode >> $HOME/.ssh/known_hosts
```

Comment:
Why is that necessary? Anyways, I'm sure it does not hurt.

Reply:
You could disable strict host key checking (otherwise you’d be prompted to confirm the connection). But I can’t imagine a reason to disable this security feature.